src/System.Windows.Forms/tests/TestUtilities/UseSystemRenderingModeAsDefault.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Windows.Forms.Primitives;
+
+namespace System;
+
+public readonly ref struct UseSystemRenderingModeAsDefault
```

Comment:
We have been using the suffix "Scope" for these types of classes.
```suggestion
public readonly ref struct UseSystemRenderingModeAsDefaultScope
```